pkg/api/feed.go

```diff
@@ -252,6 +254,14 @@ func (s *Service) feedPostHandler(w http.ResponseWriter, r *http.Request) {
 		jsonhttp.InternalServerError(ow, "done split failed")
 		return
 	}
+	// TODO: do we want to allow feed act upload/ download?
```

Comment:
The question is: do we want to allow ACT encrypted upload via the feed endpoint ? Uploading a feed is basically an soc upload but download happens via 'topic+publickey' combination. Not sure how to handle that.

Reply:
I think it can cause unexpected behavior, but can we block the user from doing that? If yes, then we probably should.

Reply:
So if the user fetches the soc address by checking it first via ACT (chunk endpoint maybe), then it should work.

Reply:
I'm a little confused here, because: feedPostHandler uploads a manifest (which work just like /bzz) but feedGetHandler return a chunk (reference), similarly to soc or chunk. If we allow a feed ACT upload, then how should that be decrypted in GET ?